### ISSUE TITLE
"jupyterlab-s3-browser" does not yet support the current version of JupyterLab.

### ISSUE THREAD

**Issue by timhughes** [IBM/jupyterlab-s3-browser#12]
just a FYI
```
$ pipenv run jupyter labextension install jupyterlab-s3-browser
Loading .env environment variables…
An error occured.
ValueError: The extension "jupyterlab-s3-browser" does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab            Extension      Package
>=4.0.2 <4.1.0        4.0.0          @jupyterlab/coreutils
See the log file for details:  /tmp/jupyterlab-debug-kwkdeu54.log
```

**Comment by vagarwal77** [comment]
Any update to this issue as i am also seeing the same issue

**Comment by reevejd** [comment]
Oops, my bad. Thanks for bringing this up. It's fixed in the newly released version 0.4.1.
